bi_view_editor/security/res_groups.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo>
```

Comment:
```suggestion
<odoo noupdate="1">
```
I think groups usually are created as not-updateable, otherwise if the user edits the group their modifications are overwritten by the module